app/js/components/searchHistory/searchHistory.css

```diff
@@ -15,28 +15,22 @@
     background-color: #ccc;
     color: #000;
     padding: 10px;
-    border-color: #000;
-    border-width: 1px;
-    border-style: solid;
     margin-top: 20px;
     margin-bottom: 20px;
 }
 
 .result-window {
     padding: 10px;
-    border-color: #000;
-    border-width: 1px;
-    border-style: solid;
```

Comment:
Is there any reason why you choose not to declare these properties using the single border property ? I'd like to learn why :)